packages/jest-benchmark/src/reporter.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+const path = require('path');
+const chalk = require('chalk');
+const readline = require('readline');
+
+const RUNNING_TEXT = ' BENCHMARK ';
+const RUNNING = chalk.reset.inverse.yellow.bold(RUNNING_TEXT) + ' ';
+
+const COMPLETE_TEXT = ' BENCHMARK ';
+const COMPLETE = chalk.reset.inverse.green.bold(COMPLETE_TEXT) + ' ';
+
+const SUCCESS = chalk.green('✓');
+const ERROR = chalk.red('✗');
```

Comment:
Ah okay, I wasn't sure here – it's not the easiest thing to regex search for. Can you link me to the source?